demos/demo/src/components/Email/index.js

```diff
@@ -1,16 +1,12 @@
 import React from 'react'
-import {connect} from 'cerebral/react'
 
-export default connect(
-  null,
-  function Phone ({email}) {
-    return (
-      <a className='level-item' target='_blank'
-        href={email && `mailto:${email}`}>
-        <span className='icon is-small'>
-          <i className='fa fa-envelope' style={email ? null : {color: '#ccc'}} />
-        </span>
-      </a>
-    )
-  }
-)
+export default function Phone ({email}) {
```

Comment:
Let us fix Firebare Demo in separate branch... to not block this one, other maintainers on it and I do not believe current master Firebase Demo runs correctly anyways